sys/kern/kern_clock.c

```diff
@@ -531,7 +534,10 @@ hardclock(int cnt, int usermode)
 			if (i > 0 && i <= newticks)
 				watchdog_fire();
 		}
-		intr_event_handle(clk_intr_event, NULL);
+#ifdef DEV_ACPI
+		if (!CK_SLIST_EMPTY(&clk_intr_event->ie_handlers))
+			intr_event_handle(clk_intr_event, NULL);
+#endif
```

Comment:
This isn't actually required for the implementation.  Issue is without this you end up with a mysterious "swi4:" interrupt producing large numbers of stray interrupts and no clue as to what is happening.